src/Apps/Collect/Components/Collection/Header/index.tsx

```diff
@@ -191,7 +247,17 @@ const BreadcrumbContainer = styled(Sans)`
   }
 `
 
-const DescriptionContainer = styled(Flex)``
+const EntityContainer = styled(Box)<{
+  isColumnLayout: boolean
+}>`
+  min-width: ${props => (props.isColumnLayout ? "" : "200px")};
```

Comment:
I think this may resolve to incorrect css when `isColumnLayout` is true (`min-width: ;`)

i would do the following instead
```
  ${p => p.isColumnLayout ? "" : "min-width: 200px;"}
```